mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/ui/SettingsActivity.kt

```diff
@@ -24,39 +24,46 @@ class SettingsActivity : BaseActivity(), SettingsView {
     @Inject
     var mPresenter: SettingsPresenter? = null
```

Comment:
This is very bad code smell, presenter can't be null. Presenter are single truth data source.